packages/backend/src/graphql/schema.graphql

```diff
@@ -89,6 +89,9 @@ type ActionSubstepArgument {
   options: [ArgumentOption]
   source: ActionSubstepArgumentSource
   dependsOn: [String]
+
+  # Only for multirow
+  fields: [ActionSubstepArgument]
```

Comment:
let's call this subfields to reduce confusion

Reply:
or columns?

Reply:
Prefer `subFields` because we're not rendering this as columns; prevents future us from being confused

Reply:
Actually on 2nd thoughts `columns` has a easier mental model

Reply:
Considered both `columns` and `subFields`, I got confused when thinking about `columns` since we don't render them horizontally. Prefer `subFields`.